devices/common_patches/hrtimv2.yaml

```diff
@@ -0,0 +1,23 @@
+HRTIM_TIMF:
+  _modify:
+    RSTE1R:
+      name: RSTF1R
+HRTIM_Common:
+  OENR:
+    _modify:
+      TF1ODS:
+        name: TF1OEN
+        description: Bit 10 - Timer F Output 1 Enable
+        _read:
```

Comment:
We keep `enumeratedValues` in `peripherals`.
Please split this file. Leave `_modify` part here and move enums in `peripherals`.

Reply:
Okay, thanks will (try to) do! :)

Reply:
I hope this better :)

Reply:
Anything at all I can do to help this move forward? :)